src/pages/index.tsx

```diff
@@ -1,8 +1,11 @@
+
 import WithPageLayout from "interfaces/with-page-layouts";
 import DefaultLayout from "components/templates/layouts/default-layout";
 import TwitterFeed from "components/organisms/twitter-feed/twitter-feed";
 import Head from "next/head";
 
+import { HeroStory } from "stories/molecules/hero.stories";
```

Comment:
Change this back to importing from the components folder. The stories are meant to document the component but aren't meant to be used in the project itself. The way you had it previously was good.